WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -114,23 +114,23 @@ I. TERMS AND CONDITIONS
 
-    The software included with the Service is open source under the MIT license and includes the following:
+    The software included with the Service is open-source under the MIT license and includes the following:
     Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
     The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
     The warranty issues covered by the MIT license will be addressed in the following section.
 
 6 DISCLAIMER OF WARRANTIES
 ==========================
 
-    Our Services are provided “as is” with no warranty of any kind. Your use of our Services is at Your sole risk, subject to the terms and conditions of liability contained in Sections 1.2 and 3. of these Terms and Conditions. 
+    Our Services are provided “as is” with no warranty of any kind. Your use of our Services is at Your sole risk, subject to the terms and conditions of liability contained in Sections 1.2 and 3. of these Terms and Conditions.
 
-    EXCEPT AS EXPRESSLY STATED IN THESE TERMS, WE DISCLAIM (TO THE FULLEST EXTENT PERMITTED BY APPLICABLE LAW) ALL WARRANTIES, REPRESENTATIONS AND CONDITIONS, WHETHER EXPRESS OR IMPLIED AND WHETHER IMPOSED BY STATUTE OR OTHERWISE, INCLUDING WITHOUT LIMITATION ANY IMPLIED WARRANTIES RELATING TO TITLE, NON-INFRINGEMENT, MERCHANTABILITY, AND FITNESS FOR A PARTICULAR PURPOSE. YOU ACKNOWLEDGE THAT YOU HAVE NOT ENTERED INTO THIS AGREEMENT IN RELIANCE UPON ANY STATEMENT, WARRANTY OR REPRESENTATION EXCEPT THOSE EXPRESSLY AND SPECIFICALLY SET FORTH IN THESE TERMS AND THAT YOU SHALL HAVE NO REMEDIES IN RESPECT OF ANY STATEMENT, WARRANTY, REPRESENTATION OR CONDITION THAT IS NOT EXPRESSLY AND SPECIFICALLY SET FORTH IN THESE TERMS. 
+    EXCEPT AS EXPRESSLY STATED IN THESE TERMS, WE DISCLAIM (TO THE FULLEST EXTENT PERMITTED BY APPLICABLE LAW) ALL WARRANTIES, REPRESENTATIONS AND CONDITIONS, WHETHER EXPRESSED OR IMPLIED AND WHETHER IMPOSED BY STATUTE OR OTHERWISE, INCLUDING AND WITHOUT LIMITATION ANY IMPLIED WARRANTIES RELATING TO TITLE, NON-INFRINGEMENT, MERCHANTABILITY, AND FITNESS FOR A PARTICULAR PURPOSE. YOU ACKNOWLEDGE THAT YOU HAVE NOT ENTERED INTO THIS AGREEMENT IN RELIANCE UPON ANY STATEMENT, WARRANTY OR REPRESENTATION EXCEPT THOSE EXPRESSLY AND SPECIFICALLY SET FORTH IN THESE TERMS AND THAT YOU SHALL HAVE NO REMEDIES IN RESPECT OF ANY STATEMENT, WARRANTY, REPRESENTATION OR CONDITION THAT IS NOT EXPRESSLY AND SPECIFICALLY SET FORTH IN THESE TERMS.
```

Comment:
GitHub shows that something is changed here, but I cannot tell what exactly...

Reply:
Before:
CONDITIONS, WHETHER EXPRESS OR IMPLIED AND WHETHER IMPOSED BY STATUTE OR OTHERWISE, INCLUDING WITHOUT LIMITATION ANY IMPLIED WARRANTIES RELATING TO TITLE
After:
CONDITIONS, WHETHER EXPRESS**ED** OR IMPLIED AND WHETHER IMPOSED BY STATUTE OR OTHERWISE, INCLUDING **AND** WITHOUT LIMITATION ANY IMPLIED WARRANTIES RELATING TO TITLE